nrf_modem/doc/pdn_extension.rst

```diff
@@ -148,7 +148,7 @@ The example code shows how to specify an APN to use for a DNS query.
 In this example, the DNS hints are extended with an extra linked-list hint that specifies the APN to use for the query.
 
 Like for data sockets, the APN name must be set up and activated before the DNS query.
-Otherwise, the query will fail.
+Otherwise, the query will fallback on the default PDN if it exist.
```

Comment:
A bit unsure about this one, this is what's happens today. Is this the correct behavior or is the old documentation describing the correct behavior and we should update the implementation?

Reply:
I would like `getaddrinfo()` to fail with `ENOTCONN` if a PDN socket does not exist for this APN, and `ENETDOWN` if the PDN socket exists but is not active.

Reply:
No idea about this question, but a language fix in case the changed version is what should be used. ;)
```suggestion
Otherwise, the query will fall back on the default PDN (if it exists).
```